swarm/network/kademlia_test.go

```diff
@@ -170,18 +170,18 @@ func TestHealthStrict(t *testing.T) {
 	// no peers
 	// unhealthy (and lonely)
 	k := newTestKademlia("11111111")
-	assertHealth(t, k, false, false)
+	// assertHealth(t, k, false, false)
```

Comment:
Is it ok to panic here? If it is, can we test it, or to remove the commented code?